src/Panel/Model.php

```diff
@@ -467,6 +459,36 @@ public function url(bool $relative = false): string
 		return $this->model->kirby()->url('panel') . '/' . $this->path();
 	}
 
+	/**
+	 * Creates an array with two versions of the content:
+	 * `latest` and `changes`.
+	 *
+	 * The content is passed through the Fields class
+	 * to ensure that the content is in the correct format
+	 * for the Panel. If there's no `changes` version, the `latest`
+	 * version is used for both.
+	 */
+	public function versions(): array
+	{
+		$language = Language::ensure('current');
+		$fields   = Fields::for($this->model, $language);
+
+		$latestVersion  = $this->model->version('latest');
+		$changesVersion = $this->model->version('changes');
+
+		$latestContent  = $latestVersion->content($language)->toArray();
+		$changesContent = $latestContent;
```

Comment:
`Discuss:` it feels wrong that we send a changes content along even when there is none. Shouldn't we rather send `null` along in this case?

Reply:
Then the frontend would need to take care of the fallback. I'm fine with that too. It just felt a bit more pragmatic - and maybe a bit more lazy :)

Reply:
I guess it has been the same before this PR, right? Cause `Form::for()` merged in the latest version even for `$model->changes()`.

Reply:
Yes, but we no longer want the merging part. That's the big difference now. We can finally prevent the merging logic at this point.